packages/components/input/src/input.ts

```diff
@@ -58,6 +58,13 @@ export const inputProps = buildProps({
     type: String,
     default: 'text',
   },
+  /**
+   * @description rows of input
+   */
+  rows: {
+    type: Number,
+    default: 1,
+  },
```

Comment:
There is no need to set the rows prop here. The native textarea supports setting rows directly, and attrs is also directly bound in the code.

Reply:
tanks a lot. this problem may exist the ide and my version of ts